docs/source/index.md

```diff
@@ -5,103 +5,171 @@
-   $ python -m pip install --use-feature=truststore Django
+# Install Django using system trust stores
+$ python -m pip install --use-feature=truststore Django
 ```
 
 This requires `truststore` to be installed in the same environment as the one running pip and to be running Python 3.10 or later. For more information you can [read the pip documentation about the feature](https://pip.pypa.io/en/stable/user_guide/#using-system-trust-stores-for-verifying-https).
 
 ### Using truststore with urllib3
 
-This `SSLContext` works the same as an {py:class}`ssl.SSLContext`.
-You can use it anywhere you would use an {py:class}`ssl.SSLContext` and
-system trust stores are automatically used to verify peer certificates:
+```{code-block} python
+import urllib3
+import truststore
+
+truststore.inject_into_ssl()
+
+http = urllib3.PoolManager()
+resp = http.request("GET", "https://example.com")
+```
+
+If you'd like to use the `truststore.SSLContext` directly you can pass
+the instance via the `ssl_context` parameter:
 
 ```{code-block} python
-   import ssl
-   import urllib3
-   import truststore
+import ssl
+import urllib3
+import truststore
 
-   ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
 
-   http = urllib3.PoolManager(ssl_context=ctx)
-   resp = http.request("GET", "https://example.com")
+http = urllib3.PoolManager(ssl_context=ctx)
+resp = http.request("GET", "https://example.com")
 ```
 
 ### Using truststore with aiohttp
 
 Truststore supports wrapping either {py:class}`socket.socket` or {py:class}`ssl.MemoryBIO` which means both synchronous and asynchronous I/O can be used:
 
 ```{code-block} python
-   import ssl
-   import aiohttp
-   import truststore
+import aiohttp
+import truststore
+
+truststore.inject_into_ssl()
+
+http = aiohttp.ClientSession()
+resp = await http.request("GET", "https://example.com")
+```
+
+If you'd like to use the `truststore.SSLContext` directly you can pass
+the instance via the `ssl` parameter:
+
+```{code-block} python
+import ssl
+import aiohttp
+import truststore
 
-   ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
 
-   http = aiohttp.ClientSession(ssl=ctx)
-   resp = await http.request("GET", "https://example.com")
+http = aiohttp.ClientSession(ssl=ctx)
+resp = await http.request("GET", "https://example.com")
 ```
 
 ### Using truststore with Requests
 
+Just like with `urllib3` using `truststore.inject_into_ssl()` is the easiest method for using Truststore with Requests:
+
+```{code-block} python
+import requests
+import truststore
+
+truststore.inject_into_ssl()
+
+resp = requests.request("GET", "https://example.com")
+```
+
 Requests doesn't support passing an {py:class}`ssl.SSLContext` object to a `requests.Session` object directly so there's an additional class you need to inject the `truststore.SSLContext` instance to the lower-level {py:class}`urllib3.PoolManager` instance:
```

Comment:
Is this still needed if you did the injection above, or only if you didn't? It's not clear.

Reply:
Yeah this section is just a big hack, going to remove the whole thing. Users of requests should use global injection and be fine.